bears/java/JavaPMDBear.py

```diff
@@ -17,22 +17,14 @@ class JavaPMDBear:
     """
 
     LANGUAGES = {'Java'}
+    REQUIREMENTS = {MavenRequirement('net.sourceforge.pmd:pmd', '5.6.0'),
```

Comment:
Thanks for the suggestion @adhikasp will surely fix this. But a lot of work needs to be done first. :smiley: